scripts/metrics.py

```diff
@@ -0,0 +1,222 @@
+#####
+## Script for Snippet Metrics
+##
+## This is not meant to be fast or pretty. But it does the job and
+## it's easy to update.
+##
+##
+## Use:
+##  - ./manage.py runscript metrics --script-args 2020-01-01 2020-03-31
+##
+## Copy / Paste the results into a SpreadSheet and split text by
+## semicolon. On Google Sheets after pasting, click on the paste icon
+## at the bottom right corner of the paste area, then select "Split
+## text to columns" and finally select Semicolon.
+##
+## Example spreadsheet: https://bit.ly/2JO8Fmq
+##
+####
+import sys
+import collections
+
+from django.db.models import Sum
+
+from product_details import product_details
+
+from snippets.base.models import *
+
+
+def pprint(d):
+     print(
+          f"  Impressions; "
```

Comment:
NIT: don't need to be f-strings here or in the first print of the next function.